server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/default.ftl

```diff
@@ -30,7 +30,8 @@
         ${val}
       </#list>
     <#else>
-      <script src="/resources/external/jquery/jquery-3.6.1.js"></script>
+      <script src="/resources/external/jquery/jquery-3.7.1.js"></script>
+      <script src="/resources/external/popper/popper.js"></script>
       <script src="/resources/external/bootstrap/js/bootstrap.js"></script>
```

Comment:
We don't really need popper separate. I'm not even sure why they made it possible to separate out. It's just used internally in bootstrap, as part of bootstrap.

If you use the bootstrap bundle, the only thing you need to do is update the filename:

```suggestion
      <script src="/resources/external/bootstrap/js/bootstrap.bundle.js"></script>
```

Also, version 5.3.2 is the latest available, as of this comment.